tutorials/transformations_02_coordinate_system_manager.ipynb

```diff
@@ -9,18 +9,6 @@
     "## Imports"
```

Comment:
<p>If this tutorial is meant to replace the "transformations_0*_tutorials", I'd not place this note into the introduction. At least do not make it the first sentence. Start with "The purpose of the CSM is...".</p><p> </p><p>Would it be possible to link the CSM to its docstring inside the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/767/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Not sure what you mean. The LCS tutorial will also get an update and this on isn't meant to replace anything. Its just an update :)</p><p> </p><p>I can change the order though.</p><p> </p><p>We can link the API documentation page.</p>

Reply:
<p>Changed it.</p>